girder_worker/context/__init__.py

```diff
@@ -0,0 +1,11 @@
+from . import girder_context, nongirder_context
+
+
+def get_context():
+    try:
+        from girder.utility.model_importer import ModelImporter # noqa
+        from girder.plugins.worker import utils # noqa
+        from girder.api.rest import getCurrentUser # noqa
```

Comment:
Isn't line 6-7 enough to ensure we are in a Girder context with the worker plugin enable?

Reply:
probably,  actually i'd like to change this to check if there is an actual request object available.  Otherwise running in a virtualenv with girder causes a false positive on this test

Reply:
See: https://github.com/girder/girder_worker/issues/274